apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -351,3 +364,39 @@ const styles = StyleSheet.create({
     borderTopColor: COLORS.BORDER_GRAY,
   },
 })
+
+export const useSellInput = ({onChange}: {onChange?: () => void}) => {
```

Comment:
needed to clear unsigned tx error

Reply:
open to better ideas of course

Reply:
EditSellAmount can update `tokenId` and `quantity` so the api would be onChange this and that, in the past we will be using  a controller to manage these behaviours and would be adding everything there, I'm ok with splitting into hooks to manage it, just believe we need to be more specific in what is changing and in the api.

Reply:
My thoughts are that more hooks is always worse.

Reply:
To me more props are always worst. Definitely don't want <InputController><Input></></>

Reply:
And also that I don't get why this logic that was in a component needs to be in createOrder now. There has to be a better way.

Reply:
on top of my head, EditSellAmount was playing the controller of AmountCard, yet CreateOrder is the one that should be doing it. So we can even discard EditSellAmount

Reply:
`<FormController><FormViewYadaYada><Input1><Input2>` form controller is responsible to deal with all errors otherwise how will it know if it can proceed or not?